src/components/Hero/index.jsx

```diff
@@ -12,7 +15,7 @@ const Hero = props => (
           <h1>{props.title}</h1>
         </div>
         <div className="col-12 col-md-3 text-md-right mt-2">
-          <img src={edXLogoForBanner} alt="edX logo" />
+          <img src={edxWhiteSemiTransparentLogo} alt="edX logo" />
```

Comment:
Does the LOGO_WHITE_URL have any transparency baked in? The original white logo used here was semi-transparent and partially revealed the background so as to be more subtle than solid white.

Do we need to add transparency (through CSS) to the new white logo here?